BullsAndCowsConsole/Program.cs

```diff
@@ -0,0 +1,169 @@
+﻿using System;
+using System.Linq;
+
+
+namespace BullsAndCowsConsole
+{
+    class Program
+    {
+        static void Intro()
+        {
+            Console.WriteLine("Welcome to Bulls and Cows.");
+            Console.WriteLine("For tutorial how to play the game press F1.");
+            Console.WriteLine("Press any key to continue.");
+
+            if (Console.ReadKey().Key == ConsoleKey.F1)
+            {
+                Console.WriteLine("\nOne player, the Chooser(me), thinks of a four-digit number and the other player, the Guesser(you), tries to guess it.\n\n" +
+                    "At each turn the Guesser tries a four-digit number, and the Chooser says how close it is to the answer by giving:\n" +
+                    "The number of Bulls - correct digits in the right position.\n" +
+                    "The number of Cows - correct digits but in the wrong position.\n\n" +
+                    "The Guesser tries to guess the answer in the fewest number of turns.\n" +
+                    "The four-digit number can start with 0 and dublicates are not allowed.\n\n" +
+                    "Now give it a try.\n");
+            }
+        }
+
+        static bool IsDigitsOnly(string str)
+        {
+            int q = str.Length;
+
+            for (int i = 0; i < q; i++)
+            {
+                if (Convert.ToByte(str[i]) < 48 | Convert.ToByte(str[i]) > 57)
```

Comment:
Is this bitwise operator `|` suitable for this expression? 
Ref: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/bitwise-and-shift-operators#logical-or-operator-

Reply:
Also, how about using a string comparison here? Would this work?
```suggestion
                if (str[i] >= '0' ...
```

Reply:
Yes, point taken!
String comparison is definitely a better approach in this case.
In my current local version it is even 
`(str[i] >= '0' && str[i] <= '9')`
as I thought it would be more appropriate to seek the values within the range rather than outside the range.
I would like you to share some more information what you meant by  'Is this bitwise operator `|` suitable for this expression?' as I think I was not able to get where you are pointing at.  Did I miss something or this is exactly what meant?

Reply:
Oh, yeah, here's something that might help - https://stackoverflow.com/questions/4014535/differences-in-boolean-operators-vs-and-vs

Reply:
Okay. Now I can see my silly mistake... Thanks!